staging/script.jellyfin-fixer/resources/lib/jellyfin_fixer.py

```
@@ -0,0 +1,26 @@
+
+    Logger.start()
+
+    try:
+        kodi_monitor = KodiEventMonitor()
+        player = KodiPlayer()
+        while not kodi_monitor.abortRequested():
+            if kodi_monitor.waitForAbort(1):
+                break
+    finally:
+        Logger.stop()
+        player = None
```

Ruff (0.13.2)

Local variable `player` is assigned to but never used

Remove assignment to unused variable `player`

(F841)